tests/tests/test_mtls.py

```diff
@@ -197,7 +197,7 @@ def common_test_mtls_enterprise(self, env, algorithm=None, use_hsm=False):
             output = env.device.run("cat /etc/mender/mender.conf")
             config = json.loads(output)
             # replace mender.conf with an mTLS enabled one
-            config["ServerURL"] = "https://mtls-ambassador:8080"
+            config["ServerURL"] = "https://mtls-gateway:8080"
```

Comment:
this is on purpose mtls-gateway, right? (I have not changed that in my original PR -- in that way we just do not touch the test, which is more spectacular) but if you are changing that why not `mender-gateway`?

Reply:
> this is on purpose mtls-gateway, right? (I have not changed that in my original PR -- in that way we just do not touch the test, which is more spectacular)

Yes, that is exactly what I did inspired by your work and reported on Slack. Only later I renamed it to get the rid of the `mtls-ambassador` name.

> but if you are changing that why not `mender-gateway`?

Thanks for bringing this up. This we can discuss.

I want to avoid the name collision with the existing `mender-gateway` container name for the virtual device. If you prefer so I can set the sever one to `mender-gateway` and rename instead the client to `device-mender-gateway` or something like that.

Reply:
>I want to avoid the name collision with the existing mender-gateway container name for the virtual device. If you prefer so I can set the sever one to mender-gateway and rename instead the client to device-mender-gateway or something like that.

ok this is the naming discussion, so as long as you want to get rid of `mtls-ambassador` I do not have preference, you can keep it as is (I thought the mender-gateway on qemu is called mender-gateway-client or something like that, it is too much work to force you to change it)

Reply:
>  (I thought the mender-gateway on qemu is called mender-gateway-client or something like that, 

Yeah, that is a good guess but it is not the case. See [here](https://github.com/mendersoftware/integration/blob/f345546b75ac50e3e9744de2a3d76722b9ac8f7d/docker-compose.mender-gateway.commercial.yml#L6) and [here](https://github.com/mendersoftware/integration/blob/f345546b75ac50e3e9744de2a3d76722b9ac8f7d/docker-compose.mender-gateway.commercial.demo.yml#L6).